ecommerce/migrations/0005_product_productimage.py

```
@@ -0,0 +1,54 @@
+                ('slug', models.SlugField(help_text='SEO-friendly URL slug (auto-generated from name)', max_length=220, unique=True)),
+                ('description', models.TextField(help_text='Detailed description of the product')),
+                ('available_stock', models.PositiveIntegerField(default=0, help_text='Available stock quantity')),
+                ('status', models.CharField(choices=[('draft', 'Draft'), ('active', 'Active'), ('inactive', 'Inactive')], default='draft', help_text='Product status', max_length=10)),
+                ('approval_status', models.CharField(choices=[('pending', 'Waiting for approval'), ('approved', 'Approved'), ('rejected', 'Rejected')], default='pending', help_text='Approval status of the product by admin', max_length=10)),
+                ('created_at', models.DateTimeField(auto_now_add=True, help_text='When the product was created')),
+                ('updated_at', models.DateTimeField(auto_now=True, help_text='When the product was last updated')),
+                ('category', models.ForeignKey(help_text='Category for this product', on_delete=django.db.models.deletion.CASCADE, related_name='products', to='ecommerce.productcategory')),
+                ('user', models.ForeignKey(help_text='User who owns this product', on_delete=django.db.models.deletion.CASCADE, related_name='products', to=settings.AUTH_USER_MODEL)),
+            ],
+            options={
+                'verbose_name': 'Product',
```

Ruff (0.12.2)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)